package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nuclear-router",
-  "version": "0.0.1",
+  "version": "1.2.0",
```

Comment:
Why such a large version jump, or not 1.1.0 at least?

Reply:
I made this change based on the existing version tags (see releases for this project) of v1.0.0 and v1.1.0.  It looks like the package.json version was never updated in the past when releases were made. 😞